src/Middleware.php

```diff
@@ -60,9 +53,7 @@ public function version(Request $request)
     public function share(Request $request)
     {
         return [
-            'errors' => function () use ($request) {
-                return $this->resolveValidationErrors($request);
-            },
+            'errors' => Inertia::always($this->resolveValidationErrors($request)),
```

Comment:
this is a breaking change, isnt it? @lepikhinb

Reply:
Shouldn't be